src/footer/index.md

```diff
@@ -0,0 +1,48 @@
+---
+template: footer-nav
+email_link:
+  label: Email Us
+  email: work@keep.network
+nav_dropdowns:
+  - title: Company
+    dropdown_items:
+      - label: Whitepaper
+        url: /whitepaper
+      - label: Team
+        url: /#team
+      - label: Advisors
+        url: /#advisors
+  - title: Follow
+    dropdown_items:
+      - label: Press
+        url: /press
+      - label: Twitter
+        is_external_link: true
+        url: https://twitter.com/keep_project
+      - label: Telegram
+        is_external_link: true
+        url: https://t.me/KeepNetworkOfficial/
+      - label: Reddit
+        is_external_link: true
+        url: https://www.reddit.com/r/KeepNetwork/
+      - label: Blog
+        is_external_link: true
+        url: https://blog.keep.network/
+  - title: Community
+    dropdown_items:
+      - label: Discord
+        is_external_link: true
+        url: https://discordapp.com/invite/wYezN7v
+      - label: GitHub
+        is_external_link: true
+        url: https://github.com/keep-network/
+copyright_text: >-
+  <p>A Thesis<sup>*</sup> Build</p><p>© 2020 Keep SEZC. All Rights Reserved.</p>
```

Comment:
Could this be markdown? Makes me nervous to inline HTML, especially if we're going to be letting folks translate this like we do the tBTC site, where they can use this as a vector to inject potentially harmful HTML.

If it already is and Remark takes care of stripping bad HTML, ignore me 😬

Reply:
This definitely could be markdown! The field is set to be a markdown widget. I just wasn't sure what the markdown was for `<sup/>`, and you can't edit stuff in the admin until it gets merged to master.

Reply:
Got it! If it’s set to be markdown then I imagine we’ll get whatever protections remark gives us (hopefully good ones). Worth double checking it sanitized 😬

Reply:
`gatsby-transformer-remark` does sanitize - it relies on this [package](https://www.npmjs.com/package/sanitize-html)